### DISCUSSION TITLE
Migration error on pimcore upgrade from 10.3.4 to 10.4

### DISCUSSION THREAD

**Discussion by pdchaudhary** [pimcore/pimcore#12095] [General]
![image](https://user-images.githubusercontent.com/30948231/167858297-0bb892d4-42df-4a80-a7ba-2883b191a38c.png)

**Comment by dvesh3** [comment]
The migration **Version20211018104331** was introduced on version 10.3.0 and should be executed on 10.3.6. I'm wondering why it executes in your case when upgrading from 10.3.6

**Reply by NiklasBr** [reply]
Are you sure @dvesh3? It looks like it is this one that is failing:

https://github.com/pimcore/pimcore/commits/10.x/bundles/CoreBundle/Migrations/Version20220506103100.php

**Reply by pdchaudhary** [reply]
@dvesh3, yes but do you have any idea why it is showing this error on migration?

**Reply by dvesh3** [reply]
@NiklasBr why do you think so? @pdchaudhary sorry, not sure about this one.

**Reply by dvesh3** [reply]
@pdchaudhary you can debug this by manually executing the migration e.g. `bin/console doctrine:migrations:execute "Pimcore\Bundle\CoreBundle\Migrations\Version20211018104331"`

**Reply by NiklasBr** [reply]
Sorry, I misread the error message
